examples/plot_alignment_methods_benchmark.py

```diff
@@ -75,35 +75,19 @@
 # independent acquisitions, similar except for one acquisition parameter, the
 # encoding phase used that was either Antero-Posterior (AP) or
 # Postero-Anterior (PA). Although this induces small differences
-# in the final data, we will take  advantage of these pseudo-duplicates to
-# create a training and a testing set that contains roughly the same signals
-# but acquired independently.
+# in the final data, we will take  advantage of these pseudo-duplicates later.
 
-# The training set, used to learn alignment from source subject toward target:
-# * source train: AP contrasts for subject sub-01
-# * target train: AP contrasts for subject sub-02
-#
+# Let's organize our data into training and testing sets:
+# * Training set: Files from the first two subjects (sub-01, sub-02) with both
+#   AP and PA acquisitions. These will serve as our source subjects for
+#   alignment.
+# * Testing set: Files from the third subject (sub-04) with both AP and PA
+#   acquisitions, which will be our target subject for evaluating alignment
+#   quality.
```

Comment:
Note that we actually do want to keep a separate split between AP and PA acquisitions so we can have a "train" and a "test" data set for both our "source" subjects and "target" subjects ! So we can re-use the original PA / AP split, just now with one additional subject.